### PR TITLE
docgen: bump railroad diagram generation timeout to 120s

### PR DESCRIPTION
**Pull request by RichardJCai**
Some diagrams are too long to be generated in the current 3s. Bumping up to 120s for now since generating diagrams isn't exactly time sensitive. 

Release note: None

### CODE REVIEW

**Review by ericharmeling — APPROVED** [review]
I actually thought this timeout limit was set on the server side... This is really nice. I've been using a local JAR to build the larger files (something to also consider, for stability).

**Comment by RichardJCai** [comment]
> I actually thought this timeout limit was set on the server side... This is really nice. I've been using a local JAR to build the larger files (something to also consider, for stability).

Yeah I prefer to use the JAR as well for stability but currently docgen will fail on our generated bnfs without a JAR. Figure its beneficial to also get this working.

File: pkg/cmd/docgen/extract/extract.go [hunk 1]
```diff
@@ -77,7 +78,8 @@ func GenerateRRNet(bnf []byte) ([]byte, error) {
 	v.Add("options", "factoring")
 	v.Add("options", "inline")
 
-	resp, err := httputil.Post(context.TODO(), rrAddr, "application/x-www-form-urlencoded", strings.NewReader(v.Encode()))
+	httpClient := httputil.NewClientWithTimeout(120 * time.Second)
```
Comment (otan) [inline pkg/cmd/docgen/extract/extract.go:81]: if you want something fancy you should make this a flag with `flag.Duration` :P

Comment (RichardJCai) [reply pkg/cmd/docgen/extract/extract.go:81]: Good call I think it's pretty straight forward to do this as well. Will update.

Comment (RichardJCai) [reply pkg/cmd/docgen/extract/extract.go]: Updated PTAL @otan

**Review by ianjevans — APPROVED** [review]

**Review by otan — APPROVED** [review]

**Comment by RichardJCai** [comment]
Thanks for the reviews!
bors r+

**Pull request merged**